### PR TITLE
Enable debug info for virsh and abandon old exception rasing

### PR DESCRIPTION
**Pull request by lento-sun**
1. change all virsh.xxx(xxx) to virsh.xxx(xxx, debug=True)
2. use test.fail/error/cancel to replace old exception rasing method

sign-offed by: Yi Sun (yisun@redhat.com)

### CODE REVIEW

**Comment by lento-sun** [comment]
@will-Do , the debug now enabled, failed case with more clear debug info. pls have a review, thx

File: libvirt/tests/src/virsh_cmd/domain/virsh_managedsave.py [hunk 1]
```diff
@@ -4,7 +4,6 @@
 import logging
 
 from autotest.client import utils
```
Comment (balamuruhans) [inline libvirt/tests/src/virsh_cmd/domain/virsh_managedsave.py]: can you take this opportunity to remove autotest dependency

Comment (lento-sun) [inline libvirt/tests/src/virsh_cmd/domain/virsh_managedsave.py]: @balamuruhans thx for reviewing, changed accordingly.

File: libvirt/tests/src/virsh_cmd/domain/virsh_managedsave.py [hunk 2]
```diff
@@ -148,17 +147,18 @@ def check_multi_guests(guests, start_delay, libvirt_guests):
...
+        is_systemd = process.run("cat /proc/1/comm").stdout.count("systemd")
         if is_systemd:
             libvirt_guests.restart()
             pattern = r'(.+ \d\d:\d\d:\d\d).+: Resuming guest.+done'
         else:
-            ret = utils.run("service libvirt-guests restart | \
-            awk '{ print strftime(\"%b %y %H:%M:%S\"), $0; fflush(); }'")
+            ret = process.run("service libvirt-guests restart | \
+                              awk '{ print strftime(\"%b %y %H:%M:%S\"), \
+                              $0; fflush(); }'")
```
Comment (balamuruhans) [inline libvirt/tests/src/virsh_cmd/domain/virsh_managedsave.py]: here too, we require `shell=True`

File: libvirt/tests/src/virsh_cmd/domain/virsh_managedsave.py [hunk 3]
```diff
@@ -124,14 +123,14 @@ def check_flags_parallel(virsh_cmd, bash_cmd, flags):
         Run the commands parallel and check the output.
         """
         cmd = ("%s & %s" % (virsh_cmd, bash_cmd))
-        ret = utils.run(cmd, ignore_status=True)
+        ret = process.run(cmd, ignore_status=True)
```
Comment (balamuruhans) [inline libvirt/tests/src/virsh_cmd/domain/virsh_managedsave.py]: The code might break here, please use `shell=True`
`process.run(cmd, ignore_status=True, shell=True)`

File: libvirt/tests/src/virsh_cmd/domain/virsh_managedsave.py (hunk 2 above)
Comment (lento-sun) [inline libvirt/tests/src/virsh_cmd/domain/virsh_managedsave.py]: Oops, forgot that..  now added, thx

**Comment by balamuruhans** [comment]
Thanks @lento-sun, merge now

**Pull request merged**